packages/neon/neon_news/lib/pages/article.dart

```diff
@@ -35,7 +35,7 @@ class _NewsArticlePageState extends State<NewsArticlePage> {
 
     WidgetsBinding.instance.addPostFrameCallback((final _) async {
       if (Provider.of<NeonPlatform>(context, listen: false).canUseWakelock) {
```

Comment:
WakelockPlus also supports linux thus it can be removed from our platform interface.

Reply:
Ah nice, forgot to check that :)